site/app/controllers/UserProfileController.php

```diff
@@ -107,14 +107,18 @@ public function changePronouns() {
         $user = $this->core->getUser();
         if (isset($_POST['pronouns'])) {
             $newPronouns = trim($_POST['pronouns']);
+            $newDisplayPronouns = filter_var(trim($_POST['pronouns-forum-display']), FILTER_VALIDATE_BOOLEAN);
             //validPronouns() checks for valid option
             if ($user->validateUserData('user_pronouns', $newPronouns) === true) {
                 $user->setPronouns($newPronouns);
+                $user->setDisplayPronouns($newDisplayPronouns);
                 $user->setUserUpdated(true);
                 $this->core->getQueries()->updateUser($user);
                 return JsonResponse::getSuccessResponse([
                     'message' => "Pronouns updated successfully",
-                    'pronouns' => $newPronouns
+                    'pronouns' => $newPronouns,
+                    'before' => $newDisplayPronouns,
```

Comment:
what is the purpose of this 'before' field vs the 'display_pronouns' field?

Reply:
that was for error-checking purpose, I will remove the before field and only leave the 'pronouns' and 'display_pronouns’ fields for future use.

Reply:
please remove the before field ASAP.  thanks